test/unit/componentHandler.js

```diff
@@ -76,9 +76,9 @@ describe('componentHandler', function() {
 
   it('should upgrade a single component to an element by using its CSS classes', function() {
     var el = document.createElement('button');
-    el.className = 'mdl-button mdl-js-button';
+    el.className = 'mdl-js-button mdl-js-ripple-effect';
     componentHandler.upgradeElement(el);
-    expect($(el)).to.have.data('upgraded', ',MaterialButton');
+    expect($(el)).to.have.data('upgraded', ',MaterialButton,MaterialRipple');
```

Comment:
Perhaps you had the earlier version, which worked, and then in the commit making jsClass optional was where the bug was introduced and the bug would only occur when jsClass is omitted and multiple classes are upgraded. So probably most people haven't seen it.